class/practice2.1/main.cpp

```diff
@@ -0,0 +1,59 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <unordered_map>
+
+void loadStatistics(std::unordered_map<std::string, int>& stats) {
```

Comment:
It is better to encapsulate data into structure with methods. That will we some database class that will be constructed from filePath and the API will be load save data, perform delete user, increment user stats, clear db.
That will make your code more structured